### DISCUSSION TITLE
Example sentence for 鼻を明かす is weird/confusing

### DISCUSSION THREAD

**Discussion by Morgawr** [Jitendex/Jitendex#41] [Errors and Corrections]
JP: あの臆病者の鼻をあかしてやった。

EN: I've triumphed over that coward.

---

While I can *kinda* see where the example sentence is going and is not *completely* wrong... it's not very good. 鼻を明かす as an expression is closer to getting the advantage over someone else by surprising/outwitting them. "Triumph over" someone means to achieve a victory or to defeat someone and it can easily give the wrong impression of what the Japanese expression really means. 

If there's a better example sentence to use in the corpus, it might be a good idea to swap this one out. Otherwise it might make sense to change it to something closer to "I've outsmarted that coward" (but also the inclusion of "coward" in this new sentence feels weird/unnecessary too...)

**Comment by stephenmk** [comment]
That is currently a priority-flagged sentence for 鼻を明かす and 臆病者. I don't think it's a great example for either term. I have removed the flags for both of them.

> 231191
> あの臆病者の鼻をあかしてやった。
> 68558
> I've triumphed over that coward.

I added a priority flag for 鼻を明かす to sentence 221045

> 221045
> この新しいブーツでみんなの鼻を明かしてやりたいんだ。
> 58367
> With these new boots of mine, I hope to knock them dead. 

And a priority flag for 臆病者 to sentence 4922

> 4922
> 「実を言うと、俺、高所恐怖症なんだ」「臆病者っ！」
> 1511
> "To tell you the truth, I am scared of heights." "You are a coward!" 

CC @jimmubreen

**Reply by JMdictProject** [reply]
Thanks.
